tedana/decomposition/eigendecomp.py

```diff
@@ -11,13 +11,68 @@
 from tedana import model, utils
 from tedana.decomposition._utils import eimask, dwtmat, idwtmat
 from tedana.selection._utils import (getelbow_cons, getelbow_mod)
+from tedana.due import due, BibTeX
 
 LGR = logging.getLogger(__name__)
 
 F_MAX = 500
 Z_MAX = 8
 
 
+@due.dcite(BibTeX(
+    """
+    @inproceedings{minka2001automatic,
+      title={Automatic choice of dimensionality for PCA},
+      author={Minka, Thomas P},
+      booktitle={Advances in neural information processing systems},
+      pages={598--604},
+      year={2001}
+    }
+    """),
+    description='Introduces method for choosing PCA dimensionality '
+                'automatically')
+def _run_mlepca(data):
+    """
+    Run PCA with automatic choice of dimensionality.
+    """
+    from sklearn.decomposition import PCA
+    ppca = PCA(n_components='mle', svd_solver='full')
+    ppca.fit(data)
+    v = ppca.components_
+    s = ppca.explained_variance_
+    u = np.dot(np.dot(data, v.T), np.diag(1. / s))
+    return u, s, v
+
+
+def run_svd(data, mlepca=True):
```

Comment:
A more general question here -- we don't call this with `mlepca=False` anywhere in the codebase. What do you think is the benefit of keeping this flag, if any ? I'm genuinely unsure...

Reply:
I noticed the same thing too. I just assumed that we would support `mlepca=False` with a new argument at the command line at some point, but I don't know if it's worth it.

Reply:
I don't think so, personally. At this point honestly I'm worried we'll need to add more dimensionality constraints rather than fewer (which the `np.linalg.svd` would point towards). How would you feel about just getting rid of that option ? Sorry for all of the iterating, here !

Reply:
Happy to.